services/worker-manager/src/util.js

```diff
@@ -50,3 +50,21 @@ const createCredentials = (worker, expires, cfg) => {
   });
 };
 exports.createCredentials = createCredentials;
+
+const PAYLOAD_SENSITIVE_KEYS = ['staticSecret', 'secret', 'accessToken', 'password'];
```

Comment:
It might be confusing to see this later -- can you add a comment that these were added just-in-case, even though they are not allowed in the schema?  If the worker is sending an access token to worker-manager, we have worse design issues than just logging that information!